app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1376,6 +1376,21 @@ def _validate_sticky_enrollment(self, data):
 
         return data
 
+    def _validate_rollout_version_support(self, data):
+        if not self.instance or not self.instance.is_rollout:
+            return data
+
+        min_version = NimbusExperiment.Version.parse(self.instance.firefox_min_version)
+        rollout_version_supported = NimbusExperiment.ROLLOUT_SUPPORT_VERSION.get(
```

Comment:
So this is the tiniest nit that ever nitted but here comes my speech.

So in Python there's two ways to look up a key in a dict, either with `a_dict['key']` or `a_dict.get('key')`, but there's an important behaviour difference between the two.  If the key doesn't exist in the dict, then the square brackets will raise a `KeyError`, whereas `.get()` will return `None`.  

So often times people might use the two interchangeably, but this failure case difference will matter later when something breaks either intentionally or unintentionally, ie do you expect not to be able to find the key?  If you expect to be able to find the key, then you want to use square brackets and if the key isn't found then something is actually wrong and you want that to fail and throw to Sentry so you know.  If you expect to possibly not find the key, you do want to use `.get()` and get `None` back, and then explicitly check for it when you use the value.

So in this case since there's a finite number of keys (ie the values of the enum) and we've explicitly setup the dict with each of those enum values as keys, I'd expect that we always expect to find the key, in which case if the key isn't found we want to throw.  If we use `.get()` here and it returns `None`, then by not checking for `None` and just using the value on the following line throwing it to `parse` then that would fail but it would be masking the actual failure, that we failed a key lookup we expected to pass.

Anyways that's my long rant.

Reply:
ooooooo that's so good to know! ty!!

Reply:
heh heh heh